semantic-python/src/Language/Python/Core.hs

```diff
@@ -258,14 +259,18 @@ instance Compile Py.GeneratorExpression
 instance Compile Py.GlobalStatement
 
 instance Compile Py.Identifier where
-  compileCC Py.Identifier { text } _ = pure . pure . Name $ text
+  compile Py.Identifier { text } _ = pure . pure . Name $ text
 
 instance Compile Py.IfStatement where
-  compileCC it@Py.IfStatement{ condition, consequence, alternative} cc =
-    locate it =<< (if' <$> compile condition <*> compileCC consequence cc <*> foldr clause cc alternative)
-    where clause (R1 Py.ElseClause{ body }) _ = compileCC body cc
+  compile it@Py.IfStatement{ condition, consequence, alternative} cc =
+    locate it =<< (if'
+                    <$> compile condition (pure none)
+                    <*> compile consequence cc
+                    <*> foldr clause cc alternative
+                  )
```

Comment:
1. Parens are redundant; `=<<` is infixr 1.

2. Suggest dedenting the `<$>` and `<*>` lines to only be one level of indentation greater than the `locate` line.